src/App.tsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type Movies = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function findMovies(movies: Movies[], input: string) {
+    return movies.filter(
+      ({ title, description }) => {
+        if (input === '') {
+          return movies;
+        }
+
+        const regex = new RegExp(input.trim(), 'ig');
+
+        return (
+          title.match(regex)
+          || description.match(regex)
+        );
+      },
+    );
+  }
```

Comment:
As I see findMovies hasn`t any dependencies from App component. So you can move it outside App

Reply:
Thanks for the review! I also moved findMovies into separate file, so component looks clearer